src/content/hocuspocus/getting-started/sponsor.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Sponsor Hocuspocus
+meta:
+  title: Sponsor Hocuspocus | Tiptap Collaboration Docs
+  description: Sponsor Hocuspocus
+  category: Hocuspocus
+---
+
+## Introduction
+
+To deliver a top-notch developer experience and user experience, we put hundreds of hours of unpaid work into Hocuspocus. Your funding helps us to make this work more and more financially sustainable. This enables us to provide helpful support, maintain all our packages, keep everything up to date, and develop new features and extensions for Hocuspocus.
+
+Give back to the open source community and [sponsor us on GitHub](https://github.com/sponsors/ueberdosis)! ♥
+
+## Your benefits as a sponsor 💖
+
+- Your issues and pull requests get a `sponsor ♥` label
+- Get a sponsor badge in all your comments on GitHub
+- Invest in the future of Hocuspocus
+- Give back to the open source community
+- Show support in your GitHub profile
+
+Sounds good? [Sponsor us on GitHub!](https://github.com/sponsors/ueberdosis)
+
+## The maintainers of Hocuspocus
+
+If you are thankful for Hocuspocus, you should say thank you to the lovely people at [überdosis](https://ueberdosis.io), the company that builds this software.
+
+AND you should definitely hire us if you want us to design and build an amazing digital product for you. Bonus points if it’s somehow text editing related.
+
+## Frequently asked questions
+
+### I can’t use GitHub. How can I support you?
+
+If you’re a company, don’t want to use GitHub, don’t have a credit card or want a proper invoice from us, just reach out to us at [humans@tiptap.dev](mailto:humans@tiptap.dev).
+
+We have an [OpenCollective](https://opencollective.com/tiptap), which allows you to send money through transfer, PayPal or credit card. Donations are tax deductible for US companies.
+
+### I want consulting. What’s your rate?
+
+If you have an issue, a question, want to talk something through or anything else, [please use GitHub issues](https://github.com/ueberdosis/hocuspocus/issues) to keep everything accessible to the whole community. For everything else, reach out to [humans@tiptap.dev](mailto:humans@tiptap.dev). We can take on a limited number of custom development and consulting contracts.
```

Comment:
Consider linking the "Contact sales" page instead of our mail: https://tiptap.dev/contact-sales